builders/pallets-precompiles/pallets/proxy.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Proxy Pallet
+description: Learn about the available extrinsics, storage methods, and constants in the proxy pallet on Moonbeam.
+keywords: proxy, substrate, moonbeam, polkadot
+---
+
+# The Proxy Pallet
+
+![Proxy Moonbeam Banner](/images/builders/pallets-precompiles/pallets/proxies-banner.png)
+
+## Introduction {: #introduction }
+
+Proxy accounts can be set up to perform a limited number of actions on behalf of users and are useful for keeping the underlying accounts safe. They allow users to keep their primary account secured safely in cold storage while enabling the proxy to actively perform functions and participate in the network with the weight of the tokens in the primary account.
+
+[Substrate's proxy pallet](https://wiki.polkadot.network/docs/learn-proxies){target=_blank} enables you to create proxy accounts, remove proxy accounts, make calls as a proxy account, and make announce proxy transactions. To add and remove proxy accounts, you can use the proxy precompile: a Solidity interface that can be interacted through the Ethereum API. For more information on how to use this contract, please refer to the [Proxy Precompile](/builders/pallets-precompiles/precompiles/proxy){target=_blank} guide.
+
+This page will provide an overview of the extrinsics, storage methods, and getters for the pallet constants available in the proxy pallet.
+
+## Proxy Pallet Interface {: #proxy-pallet-interface }
+
+### Extrinsics {: #extrinsics }
+
+The proxy pallet provides the following extrinsics (functions):
+
+- **addProxy**(delegate, proxyType, delay) - registers a proxy account for the sender that is able to make calls on the sender's behalf. If delay is set to a value greater than 0, the proxy account will have to announce a transaction and wait that value of blocks before attempting to execute it as a proxy. Emits a `ProxyAdded` event
+- **announce**(real, callHash) - registers an announcement of a proxy transaction by proxy accounts that require a delay. Emits an `Announced` event
+- **anonymous**(proxyType, delay, index) - creates a new account that cannot be accessed due to the private key being inaccessible. The sender will become a proxy for the account based on the type and delay specified. Be careful, as if the proxy is removed, the primary account will not be accessible. Emits an `AnonymousCreated` event
+- **killAnonymous**(spawner, proxyType, index, height, extIndex) - removes a previously spawned anonymous proxy
```

Comment:
just curious what is the `extIndex` param?

Reply:
Tbh not quite sure since I wasn't able to test this stuff out on Moonbase Alpha. After searching about on GitHub, this is what I found:

https://github.com/paritytech/txwrapper-core/blob/f245c7a8aa15d5ef4e6886c35722c0c3be848b55/docs/interfaces/txwrapper_substrate_src._internal_.ProxyKillAnonymousArgs.md#extindex

"The extrinsic index in which the call to anonymous was processed."